R/pmcmc_state.R

```diff
@@ -220,11 +225,8 @@ pmcmc_state <- R6::R6Class(
           private$update_history_nested()
         }
 
-        if (i %% 2) {
-          private$run_fixed()
-        } else {
-          private$run_varied()
-        }
+        run_alternate_step(j)
+        j <- j + 1
```

Comment:
Do we need `j` now? `i` seems like it'd be the natural choice to pass in here and will prevent weirdness around restarting changing the effective ratio